src/elements/Overlay/Overlay.tsx

```diff
@@ -31,10 +44,34 @@ export class Overlay extends BaseHTMLElement<IRunnerOverlayAccessor> implements
         super()
         this.attachShadow({ mode: 'open' })
     }
+
     initialize() {
         while (this.childNodes.length > 0) {
             this.shadowRoot?.appendChild(this.childNodes[0]);
         }
+
+        window[baatSymbol].addEventListener(BAATEvent.HighlightElement, ((e: CustomEvent<HighlightElement>) => {
+            const bb = getBoundingBox(e.detail.element);
+
+            const highlight = document.createElement('div');
+            highlight.classList.add('highlight');
+            highlight.style.position = 'absolute';
+            highlight.style.left = `${bb.x}px`;
+            highlight.style.top = `${bb.y}px`;
+            highlight.style.width = `${bb.w}px`;
+            highlight.style.height = `${bb.h}px`;
+
+            this.shadowRoot?.appendChild(highlight);
+            window.setTimeout(() => {
+                highlight.classList.add('blink');
+            }, 0);
+            window.setTimeout(() => {
+                highlight.classList.remove('blink');
+            }, 750);
+            window.setTimeout(() => {
+                highlight.remove();
+            }, 2000);
```

Comment:
When the border is also animated (as mentioned above), the element should also be removed when the animation is done (750+750 = 1500 ms).